corehq/apps/email/models.py

```diff
@@ -0,0 +1,29 @@
+from django.db import models
+
+from corehq.motech.const import PASSWORD_PLACEHOLDER, ALGO_AES
+from corehq.motech.utils import b64_aes_decrypt, b64_aes_encrypt
+
+
+class EmailSettings(models.Model):
+    domain = models.CharField(max_length=255, unique=True)
+    username = models.CharField(max_length=255)
+    password = models.CharField(max_length=255)
+    server = models.CharField(max_length=255)
+    port = models.IntegerField(default=0)
+    from_email = models.EmailField()
+    use_this_gateway = models.BooleanField(default=False)
+    use_tracking_headers = models.BooleanField(default=False)
+    sns_secret = models.CharField(max_length=100)
+
```

Comment:
I think we will also need a field for "[SES Configuration Set Name](https://docs.aws.amazon.com/ses/latest/dg/using-configuration-sets-in-email.html)".

Reply:
Added field `ses_config_set_name`